templates/demo/web3js-react-dapp-demo/src/AccountDetail.tsx

```diff
@@ -0,0 +1,283 @@
+import {
+  type ChangeEvent,
+  type FormEvent,
+  type MutableRefObject,
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from "react";
+
+import { v4 as uuid } from "uuid";
+
+import type { SendTransactionEvents, NewHeadsSubscription } from "web3-eth";
+
+import { type IWeb3Context, Web3Context } from "./web3/Web3Context";
+import type {
+  DataFormat,
+  providers,
+  TransactionReceipt,
+  Web3PromiEvent,
+} from "web3";
+
+function AccountDetail({ address }: { address: string }) {
+  const web3Context: IWeb3Context = useContext(Web3Context);
+
+  const [transferTo, setTransferTo] = useState<string>("");
+  const [transferAmount, setTransferAmount] = useState<number>(NaN);
```

Comment:
When dealing with wei, we don't want the value to be of type `number` due to the easy possibility of numbers being [overflowed](https://www.geeksforgeeks.org/javascript-program-to-handle-number-overflow/) due to javascript only supporting up to 2^53-1. changing this to String or bigint would be great